docs/releases/upcoming/247.doc.rst

```diff
@@ -0,0 +1 @@
+Replace TraitPrefixMap with PrefixMap in documentation example (#247)
```

Comment:
I am in two minds whether to skip this as it is quite minor.

Reply:
The news fragment wasn't included in #244, I think I'm of the view it is probably unneeded here

It is perfectly fine with me if you wish to keep it though

Reply:
Yep, I will remove it.